### ISSUE TITLE
Different price in BUY log and PAIRS

### ISSUE THREAD

**Issue by jokaorgua** [taniman/profit-trailer#2099]
NO NEW FEATURES! Use [PT Enahancements](https://github.com/taniman/profit-trailer-enhancements) for Feature Requests.

## ProfitTrailer info
Version: 2.1.17

 
Exchange:
- [ ] Bittrex
- [x] Binance
- [ ] Poloniex

 
Market:
- [x] BTC
- [ ] ETH
- [ ] USDT
- [ ] BNB
- [ ] XMR

 
Operating System:
- [ ] Windows (please specify version)
- [x] Ubuntu (please specify version)
- [ ] Other linux distro (please specify)
- [ ] Mac OS

## Expected Behavior
The price should be the same

## Current Behavior
Price is different

## Troubleshooting
Do not know how to fix this

## Steps to Reproduce
Just bought coin manually

## Log files/console output
<img width="1590" alt="2018-10-21 11 49 25" src="https://user-images.githubusercontent.com/5228422/47265390-ae544800-d527-11e8-8c3f-be776a857e5c.png">
<img width="1600" alt="2018-10-21 11 49 35" src="https://user-images.githubusercontent.com/5228422/47265391-ae544800-d527-11e8-9e99-731286e4bbbc.png">

2018-10-20 22:09:55 INFO ProfitTrailerService - Test mode: ["MANUAL"] bought 1862.93000000 RLCBTC price: 0.00007110, cost: 0.13245432
2018-10-20 22:09:55 INFO TelegramService - TEST MODE - Telegram message sent - TEST My BTC Bot
Just bought: RLCBTC
Cost: 0.13240030
Rate: 0.00007107

**Comment by taniman** [comment]
The difference you are seeing is because of fees.

you bought 1862.93 coins but paid 0.1% fee so you are left with less coins 1859.2
if we were to just sell these coins at the same bought price you would lose money.

So what we do is we make the bought_price higher to compensate.
This logic will be changed sometime in the future. 

tl;dr; This is caused by fees and it not wrong
